src/pyodide/internal/python.js

```diff
@@ -277,8 +294,44 @@ function simpleRunPython(emscriptenModule, code) {
   }
 }
 
-function mountLib(pyodide) {
-  const [info, _] = parseTarInfo();
+function buildSitePackages(tarInfo, requirements) {
+  const res = {
+    children: new Map(),
+    mode: 0o777,
+    type: 5,
+    modtime: 0,
+    size: 0,
+    path: "",
+    name: "",
+    parts: [],
+  };
+
+  for(const req of new Set([...STDLIB_PACKAGES, ...requirements])) {
```

Comment:
Nit
```suggestion
  for (const req of new Set([...STDLIB_PACKAGES, ...requirements])) {
```

Reply:
Run `npx prettier -w .` to format everything.